src/main/java/com/cdancy/bitbucket/rest/features/LabelsApi.java

```diff
@@ -45,4 +47,12 @@
     @Fallback(BitbucketFallbacks.LabelsOnError.class)
     @GET
     LabelsPage list(@Nullable @QueryParam("prefix") String prefix);
+
+    @Named("labels:get-by-name")
+    @Documentation({"https://docs.atlassian.com/bitbucket-server/rest/6.0.0/bitbucket-rest.html#idp88"})
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/api/{jclouds.api-version}/labels/{labelName}")
+    @Fallback(BitbucketFallbacks.LabelsByNameOnError.class)
+    @GET
+    Labels getLabelByName(@Nullable @PathParam("labelName") String labelName);
```

Comment:
@shinusuresh should we allow `@Nullable` here? What happens if a user passes in null?

Reply:
@cdancy Its, not mandatory as per the spec. 
I think it needs to be handled. I am updating PR

Reply:
@cdancy Is it a `NullPointerException` expected if the user passes in null?

Reply:
Can we just remove the @Nullable annotation and force users to pass in the parameter? Feels odd to have it here.

Also ... can we rename the `Labels` object to just `Label` as that would better represent what is being returned. Pluralizing it gives the impression that potentially multiple labels are getting returned.